packages/cli/src/commands/update.ts

```diff
@@ -1,64 +0,0 @@
-import chalk from 'chalk';
-
-import cmd from '../util/output/cmd';
-import logo from '../util/output/logo';
-import handleError from '../util/handle-error';
-import getArgs from '../util/get-args';
-import Client from '../util/client';
-import getUpdateCommand from '../util/get-update-command';
-import { getPkgName, getTitleName } from '../util/pkg-name';
-
-const help = () => {
-  console.log(`
-  ${chalk.bold(`${logo} ${getPkgName()} update`)} [options]
-
-  ${chalk.dim('Options:')}
-
-    -h, --help                     Output usage information
-    -d, --debug                    Debug mode [off]
-    -c ${chalk.bold.underline('NAME')}, --channel=${chalk.bold.underline(
-    'NAME'
-  )}        Specify which release channel to install [stable]
-    -r ${chalk.bold.underline('VERSION')}, --release=${chalk.bold.underline(
-    'VERSION'
-  )}  Specfic version to install (overrides \`--channel\`)
-    -y, --yes                      Skip the confirmation prompt
-
-  ${chalk.dim('Examples:')}
-
-  ${chalk.gray('–')} Update ${getTitleName()} CLI to the latest "canary" version
-
-      ${chalk.cyan(`$ ${getPkgName()} update --channel=canary`)}
-  `);
-};
-
-export default async function main(client: Client): Promise<number> {
-  let argv;
-  const { output } = client;
-
-  try {
-    argv = getArgs(client.argv.slice(2), {
-      '--channel': String,
-      '-c': '--channel',
-      '--release': String,
-      '-V': '--release',
-      '--yes': Boolean,
-      '-y': '--yes',
-    });
-  } catch (err) {
-    handleError(err);
-    return 1;
-  }
-
-  if (argv['--help']) {
-    help();
-    return 2;
-  }
-
-  output.log(
-    `Please run ${cmd(
-      await getUpdateCommand()
```

Comment:
This might be kind of useful because the update command here determines if you installed with npm or yarn, which is not that easy for the user to discover, right?

Reply:
Considering most users don't know this command even exists, I don't the command is called often.

The case of which package manager command to use is covered by the update notifier, furthermore once https://linear.app/vercel/issue/VCCLI-156/cli-update-notification-should-support-pnpm is implemented.